src/app/mail/mail-detail/mail-detail.component.ts

```diff
@@ -123,6 +129,7 @@ export class MailDetailComponent implements OnInit, OnDestroy {
       subject: mail.subject,
       parentId: this.mail.id
     };
+    this.composeMailData[mail.id].cc = this.composeMailData[mail.id].cc.filter(email => email !== this.currentMailbox.email);
```

Comment:
you only filtered cc, why not `bcc` and `to`?

Reply:
Because we only have `cc` and `to` is the `sender` of original email

Reply:
okay okay, thanks for explaination.